certbot-dns-google/certbot_dns_google/dns_google.py

```diff
@@ -183,3 +204,22 @@ def _find_managed_zone_id(self, domain):
 
         raise errors.PluginError('Unable to determine managed zone for {0} using zone names: {1}.'
                                  .format(domain, zone_dns_name_guesses))
+
+    @staticmethod
+    def get_project_id():
+        """
+        Query the google metadata service for the current project ID This only works on
+        Google Cloud Platform and raises a ServerNotFoundError if metadata.google.internal
+        is not available
+        :raises: ValueError, ServerNotFoundError
```

Comment:
Unfortunately, this line is causing formatting issues in the documentation. If you followed your our [developer installation instructions](https://certbot.eff.org/docs/contributing.html#running-a-local-copy-of-the-client) and have activated your virtual environment, you can build the documentation yourself by running `make -C docs clean html` from the `certbot/certbot-dns-google` directory.

To fix the error, the syntax here should be:
```
:raises ExceptionType: description of when it is raised
```
This is syntax is described [here](http://www.sphinx-doc.org/en/stable/domains.html#info-field-lists). There should be one line for each exception type.